lib/src/utils/resolveRichTextToNodes.spec.ts

```diff
@@ -398,6 +420,13 @@ describe("resolveNode", () => {
         },
       ],
     });
+
+    it("empty paragraph", () => {
+      expect(resolveNode(nodeWithEmptyParagraph)).toStrictEqual({
+        component: "li",
+        content: [""],
```

Comment:
You should maintain node-in-node here, thus `content: [""]` => `content: [{ content: "" }]` so we have valid nodes nested. See `renderNode` function, as if it recognizes the array, it goes though all the nodes inside it: https://github.com/NordSecurity/storyblok-rich-text-astro-renderer/blob/main/lib/RichTextRenderer.astro#L39

Btw, we should have this case too, so maybe we should add an extra case for it (not sure if this gives extra value):
![image](https://github.com/NordSecurity/storyblok-rich-text-astro-renderer/assets/9308445/674bafd1-b368-4b11-9344-1a8a8d3caa3d)